app/views/cookies/show.html.slim

```diff
@@ -1,107 +1,111 @@
-- content_for(:page_title)
-  = t(".title")
-= render "shared/heading", header_text: t(".title")
-p.govuk-body
-  = t(".paragraph_1_html", service_name: link_to(t("service.name"), root_path, class: "govuk-link"))
-p.govuk-body
-  = t(".paragraph_2", service_name: t("service.name"))
-p.govuk-body
-  = t(".paragraph_3_html", how_to_manage: link_to(t(".how_to_manage"),
-                                      "https://ico.org.uk/for-the-public/online/cookies",
-                                      class: "govuk-link"))
-h2.govuk-heading-l class="govuk-!-margin-top-6"
-  = t(".essential_cookies")
-p.govuk-body
-  = t(".essential_cookie_explainer")
-table.govuk-table
-  caption.govuk-table__caption
-    = t(".essential_cookies_used")
-  thead.govuk-table__head
-    tr.govuk-table__row
-      th.govuk-table__header[scope="col"]
-        = t(".name")
-      th.govuk-table__header[scope="col"]
-        = t(".purpose")
-      th.govuk-table__header[scope="col"]
-        = t(".expires")
-  tbody.govuk-table__body
-    tr.govuk-table__row
-      th.govuk-table__header[scope="row"]
-        = CookiesController::COOKIE_CHOICE_NAME
-      td.govuk-table__cell
-        = t(".saves_settings")
-      td.govuk-table__cell
-        = t(".one_year")
-    tr.govuk-table__row
-      th.govuk-table__header[scope="row"]
-        = LaaEstimateFinancialEligibilityForLegalAid::SESSION_COOKIE_NAME
-      td.govuk-table__cell
-        = t(".stores_user_provided_data")
-      td.govuk-table__cell
-        = t(".until_browser_closed")
+.govuk-grid-row
```

Comment:
You don't need this outermost div, as a div with this class is already defined in `application.html.slim` on line 82